src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -473,7 +473,7 @@ public virtual string CreateToken(string payload, SigningCredentials signingCred
         private string CreateTokenPrivate(JObject payload, SigningCredentials signingCredentials, EncryptingCredentials encryptingCredentials, string compressionAlgorithm, IDictionary<string, object> additionalHeaderClaims, string tokenType)
         {
             if (additionalHeaderClaims?.Count > 0 && additionalHeaderClaims.Keys.Intersect(JwtTokenUtilities.DefaultHeaderParameters, StringComparer.OrdinalIgnoreCase).Any())
-                throw LogHelper.LogExceptionMessage(new SecurityTokenException(LogHelper.FormatInvariant(LogMessages.IDX14116, nameof(additionalHeaderClaims), string.Join(", ", JwtTokenUtilities.DefaultHeaderParameters))));
+                throw LogHelper.LogExceptionMessage(new SecurityTokenException(LogHelper.FormatInvariant(LogMessages.IDX14116, LogHelper.MarkAsNonPII(nameof(additionalHeaderClaims)), LogHelper.MarkAsNonPII(string.Join(", ", JwtTokenUtilities.DefaultHeaderParameters)))));
```

Comment:
>additionalHeaderClaims [](http://example.com/codeflow?start=157&length=22)

these would not contain Pii?

Reply:
This is the name of the variable not the value, the other part is just the claim names.

Reply:
thanks. i meant the claim names.

Reply:
DefaultHeaderParameters are string constants - JwtHeaderParameterNames.Alg, JwtHeaderParameterNames.Kid, JwtHeaderParameterNames.X5t, JwtHeaderParameterNames.Enc, JwtHeaderParameterNames.Zip. Should we leave them classified as PII instead?

Reply:
Can't see how the claim names would be pii by any definition. That said, my previous comments from the SAL version of this PR stand. Better to be overly conservative and get the initial big PR asap to save yourself the ongoing merge headache.